app/assets/javascripts/measure_selection.js

```diff
@@ -269,6 +268,7 @@ ready_run_once = function() {
       }
       else if (edition == '2015') {
         setCheckboxDisabled('#product_duplicate_records', false);
+        $('#product_duplicate_records').prop("checked", true);
```

Comment:
We also need to add the logic so that this Duplicate Records box is only checked if C2 is checked, and is unchecked if C2 is unchecked, as discussed between myself, @rbclark and @dczulada, before pulling in.